queries/haskell/highlights.scm

```diff
@@ -0,0 +1,89 @@
+(integer) @number
+(exp_literal (float)) @float
+
+(char) @character
+
```

Comment:
I don't know a lot about tree-sitter, just trying this out and noticed strings were not highlighted, looking at the haskell grammar it looks like it defines a string token, should there be a 

```
(string) @string
``` 
here?